packages/blade/src/components/Accordion/__tests__/Accordion.native.test.tsx

```diff
@@ -0,0 +1,195 @@
+import { fireEvent, waitFor } from '@testing-library/react-native';
+import { useState } from 'react';
+import { Accordion } from '../Accordion';
+import { AccordionItem } from '../AccordionItem';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+import { Button } from '~components/Button';
+
+describe('<Accordion />', () => {
+  // Accordion uses Collapsible which uses animations and requestAnimationFrame which makes RN tests throw warnings
```

Comment:
🤔 So does js automatically mocks requestAnimationFrame in RN?

Reply:
[Yes](https://jestjs.io/docs/jest-object#jestusefaketimersfaketimersconfig) it seems, since we're using JSDOM env